### PR TITLE
Arm64 bitness

### LINKED ISSUE #11068: arm64 version string
### Description of the Issue
From code inspection arm64 is missing in the platform information and therefore also the version string doesn't contain 64bit attribute.

See https://docs.microsoft.com/en-us/windows/win32/api/sysinfoapi/ns-sysinfoapi-system_info

### Steps to Reproduce the Issue
Check debug info text

### Expected Behavior
OS Name : Windows 10 Pro (64-bit)

### Actual Behavior
OS Name : Windows 10 Pro (Unknown-bit)

### Debug Information
N/A

### PR DESCRIPTION
**Pull request by chcg**
Fix #11068

### CODE REVIEW

File: PowerEditor/src/Parameters.cpp [hunk 1]
```diff
@@ -791,6 +791,10 @@ winVer NppParameters::getWindowsVersion()
 		_platForm = PF_X86;
 		break;
 
+	case PROCESSOR_ARCHITECTURE_ARM64:
```
Comment (donho) [inline PowerEditor/src/Parameters.cpp:794]: It seems the symbol `PROCESSOR_ARCHITECTURE_ARM64` is not recognized by MinGW32 :(

Comment (ArkadiuszMichalski) [reply PowerEditor/src/Parameters.cpp:794]: Only [new version](https://sourceforge.net/p/mingw-w64/mingw-w64/ci/master/tree/mingw-w64-headers/include/winnt.h#l4877) has it. Maybe add it to [gcc-fixes.h](https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/PowerEditor/gcc/gcc-fixes.h)?

Comment (chcg) [reply PowerEditor/src/Parameters.cpp:794]: Corrected.

**Comment by donho** [comment]
@chcg 
I have no ARM device for testing, but it seems `getWinVerBitStr()` returns the right info (ie. 64 bits) currently on DebugInfo box under ARM system: https://github.com/notepad-plus-plus/notepad-plus-plus/issues/10640

**Comment by chcg** [comment]
@donho See the issue you mentioned:
OS Name : Windows 10 Pro (Unknown-bit)

**Pull request closed without merging**